### PR TITLE
Set openai_api_type to be passed as values to leverage azure openai's default

### PR DESCRIPTION
**Pull request by kevinco26**
- **Description:** 
Recently, this was introduced: https://github.com/langchain-ai/langchain/pull/10707

However, the changes are not taking effect unless you create the AzureOpenAIEmbeddings class with parameter openai_api_type="azure".. In the code it's already getting set by default if openAI version > 1 but it's never getting accessed when the embed_documents is actually happening because it was never surfaced up
  - **Issue:** Not so much an issue but leveraging existing default code of api type of azure when used.
  - **Twitter handle:** @kevin_neum

### CODE REVIEW

File: libs/langchain/langchain/embeddings/azure_openai.py [hunk 1]
```diff
@@ -133,6 +133,7 @@ def validate_environment(cls, values: Dict) -> Dict:
                 "azure_ad_token": values["azure_ad_token"],
                 "azure_ad_token_provider": values["azure_ad_token_provider"],
                 "organization": values["openai_organization"],
+                "openai_api_type":values["openai_api_type"],
```
Comment (efriis) [inline libs/langchain/langchain/embeddings/azure_openai.py:136]:
```suggestion
                "api_type":values["openai_api_type"],
```

**Review by efriis — DISMISSED** [review]

**Comment by efriis** [comment]
Actually, looks like the v1 client doesn't accept an `api_type` param. https://github.com/openai/openai-python/blob/65e29a2efa455a06deb59e243f27796c4ca2254c/src/openai/_client.py#L43-L61

Closing because this shouldn't go in!

**Comment by kevinco26** [comment]
Good catch! Any ideas for leveraging the default value in AzureOpenAIEmbeddings that is currently set but not being leveraged? Thus making this obselete unless it's explicitly set?
https://github.com/langchain-ai/langchain/pull/10707

**Comment by efriis** [comment]
Nope! And not sure what the linked chunk size issue has to do with this. While we support both v0 and v1, it's not completely obsolete too.

**Pull request closed without merging**